packages/tokenbank_v1/contracts/base/MetaTxModule.sol

```diff
@@ -241,4 +254,24 @@ contract MetaTxModule is BaseModule
             wallets[wallet].nonce = nonce;
         }
     }
+
+    // TODO (daniel): return false in case of ERC1271 error, not throw exception
```

Comment:
@Brechtpd can you help me do this? I'm not familiar with this technique. 

> TODO (daniel): return false in case of ERC1271 error (for example the contract doesn't support 1271 at all), not throw an exception.

Reply:
I don't there's anything really new to do this. This should work:

```
bytes memory callData = abi.encodeWithSelector(
    ERC1271(signer).isValidSignature.selector,
    hash,
    extractSignature(signatures, idx)
);
(bool success, bytes memory data) = signer.staticcall(callData);
if (success && data.length == 32) {
    return (data.bytesToBytes4(0) == ERC1271_MAGICVALUE);
} else {
    return false;
}
```

Reply:
done